tests/errors/test_literal.py

```diff
@@ -1,2 +1,6 @@
+from lpython import i32
+
 def test_literal1():
-    x: i32 = 0123
\ No newline at end of file
+    x: i32 = 0123
+
+test_literal1()
```

Comment:
Please don't modify this file, it only tests for syntax error, so I think there is no need for the function call.

Also, do a clean build and update tests.
```console
$ git clean -dfx
$ ./build.sh
$ ./run_tests.py -u
```
Before performing this, commit all the previous unstaged changes.

Reply:
Hmm..this was a bit confusing too, as some lines had function calls while some did not. Should we document this also?

Reply:
If it's the AST or parser tests, most probably it wouldn't have function call.
Let's keep this in mind.

Reply:
If it's the AST or parser tests, most probably it wouldn't have function call.
Let's keep this in mind.

Reply:
Thank you very much for this information! :+1: